recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+
+git clone --depth 1 https://github.com/twosigma/beakerx.git
```

Comment:
It's temporary. I'll use Maven artifact when Beaker release it. But you're right I should stick to a specific commit not master.